### DISCUSSION TITLE
RBP with ranges

### DISCUSSION THREAD

**Discussion by jwerner15083** [CMSgov/price-transparency-guide#496] [Q&A] [answered]
I have been searching hard for a solution to this problem.  We have contracts that are wide ranging, but are reference based (% of Medicare mostly).  It makes no sense to blow out the data to a massive scale, especially when many areas are "formula based" (such as OPPS and IPPS).  An example contract here may be:  L&D case rates, IP % of Medicare, OP % of Medicare, 100% Medicare for lab, 110% MCR for DME, 140% MCR for Radiology, 125% MCR for Physician, all followed by a % of charges for non CMS values.

So it seems that 2 items need to be added (or have been and I cannot find them)
   -  Need a % of Medicare to be added.  Perhaps % of each system (IPPS, OPPS, Professional, etc).  Or a more encompassing reference value.  
   -   Need the ability to constrain that list by a range of values.  For example Radiology may be 70000-79999, etc.
   
   Also how will maintaining this be done over time?  IPPS and OPPS for example can send time based adjustments to the provider base values for any given facility.  I only see the expiration_date on the rate, which again means blowing out literally thousands of lines (or millions) for every provider rate update on each update (by an external org).  When all of that could be compressed into a simple rule based on the reference value (%) of the underlying program (CMS, etc) constrained by a range of data values.

**Comment by joseph-kendall** [comment, marked as answer]
I believe CMS said in the March webinar that we were to calculate the % of Medicare and not report it as a percentage. They want an explicit allowed amount. 

As to point two, we are using the CSTM-ALL/CSTM-00 where appropriate and expanding the codes that are in a range to individual codes where appropriate.

**Comment by caracolina** [comment]
We are facing similar issues with Medicare-based rates. In another thread CMS said to calculate a dollar amounts based on "base rate multiplied by the percentage". Considering services as you mentioned (IPPS/OPPS) that are based on many moving parts we might consider calculating rates from on a "base scenario" and use that. Is this the direction you are going as well?

**Reply by jwerner15083** [reply]
@caracolina Currently I am calculating the base rates via a function for each type of pricing.  The underlying tables from CMS are the "base" for this, and then there is a modifier applied to them (per provider/location) to detail them out.  This is working for Acute IP and professional services, but not so well (yet) for APC based services.  

For stuff like lab, DME, professional, DRG, etc...then it is simply a calculated modifier x the base value.  So say for a professional service it is the base % (150) x the cms value (determined by carrier/locality).  More complex for DRG values since there are more variables, but the formula can be determined.  

It is a lot more work, and the accuracy is not as high as it should be...but it is close enough I would say.

**Reply by hbeauvais** [reply]
Hi,

My team is at a loss with locating the Medicare fee schedules for IP and OP. Do you have a link to these fee schedules?

**Reply by jwerner15083** [reply]
There is no real "IP or OP" fee schedule.  They are formula driven processes defined by the CMS rules.  IP is fairly easy to sort out overall, but OP is vsry complex.  You might be able to find some generic data, but it is not goign to be too specific or even useful
